src/run.ts

```diff
@@ -146,6 +146,12 @@ export async function buildSecret(
 
    return secret
 }
+function mapSecretType(inputType: string): string {
+   const t = inputType.toLowerCase().trim()
+   const K8S_SECRET_TYPE_OPAQUE = 'Opaque'
```

Comment:
this const can be defined outside the function scope so it can be reused elsewhere in the code if needed